schemas/firefox-installer/install/install.1.schema.json

```diff
@@ -1,5 +1,10 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
+  "mozPipelineMetadata": {
+    "bq_dataset_family": "firefox_installer",
+    "bq_table": "install_v1",
+    "uri_scheme": "structured"
```

Comment:
I said before I didn't want to block on figuring out pipeline family vs. uri_scheme, but regardless of that this annotation is in fact ambiguous WRT firefox installer pings i.e. if we were to parse a DSMO stub installer ping based on this metadata it would fail. To correct my earlier statements, this namespace probably uses the `structured` uri scheme for full installer and `stub_installer` for DSMO pings. These happen to map to pipeline families, but uri scheme to pipeline family is not always 1:1 e.g. pioneer uses the `telemetry` uri scheme.

For this first iteration, as long as uri scheme is only used by MSG for BQ metadata generation (n.b. its documentation indicates this explicitly) this characterization is fine (side note, pioneer uses custom metadata even though its uri scheme is `telemetry`; it seems things are not quite 1:1 all over the place). We're going to want to define all of these carefully in further iterations and additions to metadata.